### PR TITLE
config data added in page.js

### PR DESCRIPTION
**Pull request by DilipGelot**
config data added in page.js

### CODE REVIEW

File: src/app/components/Banner.jsx [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
+      {videoUrl ? (
+        <video
+          autoPlay
+          muted
+          loop
+          className="absolute inset-0 w-full h-auto"
+          style={{ width: "100%", height: "85vh", objectFit: "cover" }}
+        >
+          <source src={videoUrl} type="video/mp4" />
+          Your browser does not support the video tag.
```
Comment (DashrathGelot) [inline src/app/components/Banner.jsx:15]: if not supported then we have to show image

File: src/app/components/Banner.jsx [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
+        />
+      )}
       <div className="absolute inset-0 flex flex-col justify-center items-center">
         <div className="text-center" style={{ marginTop: "25%" }}>
-          <h1 className="text-4xl font-bold font-hind text-white">
-            Your Text Here
-          </h1>
-          <p className="text-white">Lorem ipsum dolor sit amet.</p>
+          <h1 className="text-4xl font-bold font-hind text-white">{heading}</h1>
+          <p className="text-white">{paragraph}</p>
```
Comment (DashrathGelot) [inline src/app/components/Banner.jsx:28]: give proper naming not paragraph, put tagline or something

File: src/app/components/Landing.jsx [hunk 3]
```diff
@@ -3,52 +3,26 @@ import Section from "./Section";
...
-              text={serviceTiles.text}
-            />
-          ))}
-        </Grid>
-      </Section>
+      <Banner
+        videoUrl={banner.videoUrl}
+        imageUrl={banner.imageUrl}
+        heading={banner.heading}
+        paragraph={banner.paragraph}
```
Comment (DashrathGelot) [inline src/app/components/Landing.jsx:13]: pass entire banner why we need to pass seperate

File: src/app/components/Landing.jsx [hunk 4]
```diff
@@ -3,52 +3,26 @@ import Section from "./Section";
...
+      />
+      {sections.map((section, index) => (
```
Comment (DashrathGelot) [inline src/app/components/Landing.jsx:15]: This is great well done!!! 🎉

File: src/app/components/Navbar.jsx [hunk 5]
```diff
@@ -1,43 +1,39 @@
 import React from "react";
 
-const Navbar = () => {
+const Navbar = ({ logo, text }) => {
```
Comment (DashrathGelot) [inline src/app/components/Navbar.jsx:3]: I can see lot of duplicates in this file can you create separate component like IconButton and use that??

File: src/app/components/Tile.jsx [hunk 6]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 
-const Tile = ({ image, title, text }) => {
+const Tile = ({ image, title, text = "" }) => {
```
Comment (DashrathGelot) [inline src/app/components/Tile.jsx:3]: why text = "" ???

File: src/app/page.js [hunk 7]
```diff
@@ -1,12 +1,99 @@
...
+      ],
+    },
+  };
+
   return (
     <main className="overflow-x-hidden">
-      <Navbar />
-      <Landing />
-      <Footer />
+      <Navbar logo={config.navbar.logo} text={config.navbar.text} />
```
Comment (DashrathGelot) [inline src/app/page.js:94]: both are navbar just pass config.navbar why do we need separate??

File: src/app/components/Tile.jsx (hunk 6 above)
Comment (DilipGelot) [reply src/app/components/Tile.jsx:3]: i'm just checking what happend if we pass nothing in text and forgot to remove that

File: src/app/components/Banner.jsx [hunk 8]
```diff
@@ -1,24 +1,37 @@
...
-        autoPlay
-        muted
-        loop
-        className="absolute inset-0 w-full h-auto"
-        style={{ width: "100%", height: "85vh", objectFit: "cover" }}
-      >
-        <source src="https://shorturl.at/HQR14" type="video/mp4" />
-        Your browser does not support the video tag.
-      </video>
+      {videoUrl ? (
```
Comment (DashrathGelot) [inline src/app/components/Banner.jsx:7]: can you take url in props and figure out is it video or image based on extension and render based on that???

File: src/app/components/footer.jsx [hunk 9]
```diff
@@ -2,50 +2,7 @@ import React from "react";
 import Grid from "../components/common/Grid";
 import FooterLinkGroup from "./FooterLinkGroup";
 
-const Footer = () => {
```
Comment (DashrathGelot) [inline src/app/components/footer.jsx:5]: Change file name it should be "**F**ooter.jsx"

File: src/app/components/Banner.jsx [hunk 10]
```diff
@@ -1,24 +1,38 @@
 import React from "react";
 
-const Banner = () => {
+const isVideo = (url) => {
```
Comment (DashrathGelot) [inline src/app/components/Banner.jsx:3]: this function should be inside the component !!!!

**Review by DashrathGelot — APPROVED** [review]

**Pull request merged**